hurumap_tz/templates/profile/profile_detail.html

```diff
@@ -1,403 +1,540 @@
-             <div class="column-full">
-                 <div id="chart-histogram-health_centers_distribution-health_center_data" data-stat-type="scaled-percentage" data-chart-title="Number of health centers by region (2014)"></div>
-            </div>
+      </header>
+      <div class="section-container">
+        <section class="clearfix stat-row">
+          <p class="source">Source:
+            <a href="{{ health_centers_distribution.source_link }}">{{ health_centers_distribution.source_name }}</a>
+          </p>
+          <br clear="all">
+          <div class="column-third">
+            {% include 'profile/_blocks/_stat_list.html' with stat=health_centers_distribution.total stat_type='number' %}
+          </div>
+          <div class="column-third">
+            {% include 'profile/_blocks/_stat_list.html' with stat=health_centers_distribution.hiv_centers stat_type='number' %}
+          </div>
+          {% if not health_centers_distribution.health_center_ownership.is_missing %}
+            <div class="column-third" id="chart-pie-health_centers_distribution-health_center_ownership" data-initial-sort="-value" data-stat-type="scaled-percentage" data-initial-sort="-value" data-chart-title="Ownership"></div>
+
+          {% endif %} <!-- end if -->
+          <div class="column-full">
+            {% if not health_centers_distribution.health_center_data.is_missing %}
+              <div id="chart-histogram-health_centers_distribution-health_center_data" data-stat-type="scaled-percentage" data-chart-title="Number of health centers by region (2014)"></div>
+            {% endif %} <!-- end if -->
+          </div>
         </section>
-    </div>
-</article>
+      </div>
+    </article>
+  {% endif %} <!-- end if -->
 
-<article id="health_workers_distribution" class="clearfix">
-    <header class="section-contents">
+  {% if not health_workers_distribution.is_missing %}
+    <article id="health_workers_distribution" class="clearfix">
+      <header class="section-contents">
         <h1>Health workers distribution</h1>
-    </header>
-    <div class="section-container">
-         <section class="clearfix stat-row">
-             <p class="source">Source: <a href="{{health_workers_distribution.source_link}}">{{health_workers_distribution.source_name}}</a></p>
-             <br clear="all">
-             <div class="column-third">
-                {% include 'profile/_blocks/_stat_list.html' with stat=health_workers_distribution.hrh_patient_ratio stat_type='number' %}
-              </div>
-             <div class="column-full">
-                 <div id="chart-histogram-health_workers_distribution-health_worker_data" data-stat-type="scaled-percentage" data-chart-title="Number of health workers by profession(2014)"></div>
-            </div>
+      </header>
+      <div class="section-container">
+      <section class="clearfix stat-row">
+          {{ health_workers_distribution }}
+      </section>
```

Comment:
This just dumps the object.

![screenshot from 2019-01-21 10-42-50](https://user-images.githubusercontent.com/1779590/51459375-654fa200-1d69-11e9-966d-a5d95a97e506.png)

Reply:
@kilemensi that will be removed, I was debugging why charts weren't being displayed

Reply:
Do you remember which region was causing problems @phillipahereza? Because with the above two fixes, Tanzania looks great to me:

![screenshot_2019-01-21 hurumap tanzania profile tanzania](https://user-images.githubusercontent.com/1779590/51459674-5b7a6e80-1d6a-11e9-875c-d1ffb953f71b.png)


I'll try `lands` again.

Reply:
hmmm I don't remember, but probably Morogoro @kilemensi